x-pack/plugins/security_solution/public/common/components/query_bar/index.tsx

```diff
@@ -77,7 +77,9 @@ export const QueryBar = memo<QueryBarComponentProps>(
       (savedQueryUpdated: SavedQuery) => {
         const { query: newQuery, filters: newFilters, timefilter } = savedQueryUpdated.attributes;
         onSubmitQuery(newQuery, timefilter);
-        filterManager.setFilters(newFilters || []);
+        if (filterManager) {
+          filterManager.setFilters(newFilters || []);
```

Comment:
We might want to use `setAppFilters` in this context to scope them to the app. Same for the usages of `setFilters` below.

Reply:
so outside of timeline, setAppFilters, inside setFilters on it's own instance of FilterManager?

Reply:
Jep 👍